boxoffice/models/line_item.py

```diff
@@ -24,6 +24,25 @@ def make_ntuple(item_id, base_amount, **kwargs):
         kwargs.get('discounted_amount', Decimal(0)))
 
 
+class Assignee(BaseMixin, db.Model):
+    __tablename__ = 'assignee'
+    __table_args__ = (db.UniqueConstraint('item_collection_id', 'email'),)
+
+    fullname = db.Column(db.Unicode(80), nullable=False)
+    #: Unvalidated email address
+    email = db.Column(db.Unicode(254), nullable=False)
+    #: Unvalidated phone number
+    phone = db.Column(db.Unicode(80), nullable=True)
+    details = db.Column(JsonDict, nullable=False, server_default='{}')
+
+    # Track the assignee from whom the line_item was transferred from
+    parent_id = db.Column(None, db.ForeignKey('assignee.id'), nullable=True)
+    parent = db.relationship('Assignee', uselist=False)
+
+    item_collection_id = db.Column(None, db.ForeignKey('item_collection.id'), nullable=False)
+    item_collection = db.relationship(ItemCollection, backref=db.backref('assignees', cascade='all, delete-orphan'))
```

Comment:
Don't add this here. It'll break your table normalisation. Find assignees for an item collection via the Lineitem model.